CorsixTH/Lua/cheats.lua

```diff
@@ -169,3 +176,95 @@ function Cheats:cheatDecreasePrices()
     end
   end
 end
+
+--[[Begin toggle-based cheat functions]]
+
+--! Enable Roujin's challenge (spawn rate cheat)
+function Cheats:roujinOn()
```

Comment:
Are these toggles accessed outside of the table_cheats table / this file?

Reply:
No, not anymore I believe...